pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -607,6 +607,14 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("get").Groups(projectGroup).Resources("projects").RuleOrDie(),
 			},
 		},
+		{
```

Comment:
move the definition and binding to infra_sa_policy.go, and follow the pattern there... we define a service account name, role name, and permissions we expect that service account to have